keyboards/geigeigeist/klor/klor.h

```diff
@@ -0,0 +1,42 @@
+// Copyright 2024 QMK
+// SPDX-License-Identifier: GPL-2.0-or-later
+#include "quantum.h"
+
+#ifdef OLED_ENABLE
+static const char PROGMEM klor_face[] = {
```

Comment:
Having this here would cause both `keymap.c` and `klor.c` to both have a copy of the `klor_face` symbol, as well as any other file that pulls in this header.

My preference would be for this file to be removed, and `klor_face` added to `klor.c`. If the intent is to still provide a way for users to access, then as non static, or a non static `render_klor_face()` function.